app/components/mandatarissen/fractie-selector.js

```diff
@@ -85,4 +131,38 @@ export default class MandatenbeheerFractieSelectorComponent extends Component {
     let searchResults = await this.fetchFracties(searchData);
     return searchResults;
   });
+
+  async isFractieIndependent(fractie) {
+    const type = await fractie.fractietype;
+
+    if (!type) {
+      return false;
+    }
+
+    return type.isOnafhankelijk;
+  }
+
+  async getIndependentFractie(fracties) {
+    for (const fractie of fracties) {
+      const isIndependent = await this.isFractieIndependent(fractie);
+
+      if (isIndependent) {
+        return fractie;
+      }
+    }
+    return null;
+  }
+
+  get mandataris() {
```

Comment:
Personally I wouldn't do this, because it seems pretty redundant, but I can live with it.